src/lib/doctrine-dbal-bulk/src/Flow/Doctrine/Bulk/Columns.php

```diff
@@ -34,11 +34,6 @@ public function all() : array
         return $this->columns;
     }
 
-    public function concat(string $separator) : string
```

Comment:
why?

Reply:
It's not used anymore, we can leave it ofc, but it's only usage was to concat with `,` which now is done with quoting on the level of dialect classes, not in `Columns`.